build.gradle.kts

```diff
@@ -13,3 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+plugins {
+  `java-library`
+  `maven-publish`
+}
+
+repositories {
+  mavenLocal()
```

Comment:
we might not want to use this:
https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:case-for-maven-local

Reply:
I think other repositories also use it. I just followed them.

Removed as of now.